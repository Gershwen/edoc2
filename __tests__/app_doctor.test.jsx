//snapshot test of /pages/app_doctor.js

import { render } from '@testing-library/react';
import App from '../pages/app_doctor';

it('renders app_doctorpage unchanged', () => {
    const { container } = render(<App />)
    expect(container).toMatchSnapshot()
  })