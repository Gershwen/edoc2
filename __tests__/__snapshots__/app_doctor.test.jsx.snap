// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div>
    <div
      class="wrapper"
    >
      <h1>
        eDoc
      </h1>
      <button
        class="sign-in"
        variant="primary"
      >
        Sign in
      </button>
    </div>
    <div
      class="welcome-wrapper"
    >
      <h2>
        Welcome to eDoc
      </h2>
      <p>
        The application built for doctors that takes care of the paperwork so that doctors can take care of the people.
      </p>
    </div>
  </div>
</div>
`;
